src/components/Container/Container.module.css

```diff
@@ -1,14 +1,17 @@
 .wrapper {
   background-color: var(--background-accent);
-  height: calc(100vh - 70px);
```

Comment:
jeśli height jest inny niż 100vh, to na mobilce w patch currency tło nie wypełnia całej strony